obs/preinstallimage-bios.sh.in

```diff
@@ -668,7 +677,7 @@ case "$IMGTYPE" in
     *devel*)
         echo "BIOS_LOG_LEVEL=LOG_DEBUG" > /usr/share/fty/etc/default/fty
         echo "BIOS_NUT_USE_DMF=true" >> /usr/share/fty/etc/default/fty
-        sed -i "s|INFO|DEBUG|g" /etc/fty/ftylog.cfg 
+        sed -i "s|rootLogger=INFO|rootLogger=DEBUG|g" /etc/fty/ftylog.cfg
```

Comment:
not sure that's needed

Reply:
+1, whatever we defined explicitly as INFO for end-user deployments (and however it mutates later), we want DEBUGged in devel/test images :)

Reply:
In devel mode, debug logs are redirected to console only. The console appender threshold = INHERIT, and the SysLog appender threshold = INFO. Eg., debug logs are not persisted.

Reply:
"debug logs are not persisted" - is that "at all" or over (RAM-backed?) journal rotations? Because so far these were the most useful ones ;p

Reply:
For a devel image, debug logs will be only redirected to console (displayed with journalctl -f). With "journalctl -D /var/log/journal -f", you will only see INFO+ log messages. If you want to persist debug messages in your devel image, simply change SysLog.threshold to DEBUG (or INHERIT) in ftylog.cfg. The main idea is "no persistency of DEBUG or less messages in production image", to not overcharge our journald logfile with 'developer/technical info'.